sno/diff_output.py

```diff
@@ -67,20 +68,83 @@ def _out(dataset, diff):
         path = dataset.path
 
         prefix = f"{path}:meta:"
-        for key, delta in sorted(diff.get('meta', {}).items()):
+        schema_delta = None
+        for key, delta in sorted(diff.get("meta", {}).items()):
             if delta.old:
                 click.secho(f"--- {prefix}{delta.old_key}", bold=True, **pecho)
             if delta.new:
                 click.secho(f"+++ {prefix}{delta.new_key}", bold=True, **pecho)
+            if key == "schema" and delta.old and delta.new:
+                # Schema is handled below
+                schema_delta = delta
+                continue
             if delta.old:
                 click.secho(prefix_json(delta.old_value, "- "), fg="red", **pecho)
             if delta.new:
                 click.secho(prefix_json(delta.new_value, "+ "), fg="green", **pecho)
 
+        if schema_delta:
```

Comment:
Can you move the schema-diff specific stuff out of output_text and into a helper function?

Reply:
This is done